contractcourt/channel_arbitrator.go

```diff
@@ -129,13 +129,31 @@ type ChannelArbitratorConfig struct {
 	ChainArbitratorConfig
 }
 
+// ReportOutputType describes the type of output that is being reported
+// on.
+type ReportOutputType uint8
+
+const (
+	// ReportOutputIncomingHtlc is an incoming hash time locked contract on
+	// the commitment tx.
+	ReportOutputIncomingHtlc ReportOutputType = iota
+
+	// ReportOutputOutgoingHtlc is an outgoing hash time locked contract on
+	// the commitment tx.
+	ReportOutputOutgoingHtlc
+
+	// ReportOutputDirect is an uncontested output on the commitment
+	// transaction paying to us directly.
+	ReportOutputDirect
```

Comment:
nit: add type in the commit where it is first used

Reply:
It is used in this commit in `arbitratorPopulateForceCloseResp` ?

Reply:
only `ReportOutputIncomingHtlc` is used there.

Reply:
but this is no blocker ofc

Reply:
Ah, was confused by 'type' vs 'value'. Moved to next commit